src/AspNet.Security.OAuth.QuickBooks/QuickBooksAuthenticationOptions.cs

```diff
@@ -0,0 +1,40 @@
+﻿/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+using System.Security.Claims;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authentication.OAuth;
+using static AspNet.Security.OAuth.QuickBooks.QuickBooksAuthenticationConstants;
+
+namespace AspNet.Security.OAuth.QuickBooks
+{
+    /// <summary>
+    /// Defines a set of options used by <see cref="QuickBooksAuthenticationHandler"/>.
+    /// </summary>
+    public class QuickBooksAuthenticationOptions : OAuthOptions
+    {
+        // private string scopeValC2QB = "com.intuit.quickbooks.accounting";
+        // private string scopeValSIWI = "com.intuit.quickbooks.payment";
+        private string scopeValOpenId = "openid email profile phone address";
+
+        public QuickBooksAuthenticationOptions()
+        {
+            ClaimsIssuer = QuickBooksAuthenticationDefaults.Issuer;
+            CallbackPath = QuickBooksAuthenticationDefaults.CallbackPath;
+
+            AuthorizationEndpoint = QuickBooksAuthenticationDefaults.AuthorizationEndpoint;
+            TokenEndpoint = QuickBooksAuthenticationDefaults.TokenEndpoint;
+            UserInformationEndpoint = QuickBooksAuthenticationDefaults.UserInformationEndpoint;
+
+            Scope.Add(scopeValOpenId);
```

Comment:
I would expect this to be:

```suggestion
            Scope.Add("openid");
            Scope.Add("email");
            Scope.Add("profile");
            Scope.Add("phone");
            Scope.Add("address");
```

Reply:
Unfortunately, your suggestion codes of the scope are not valid. The server will throw an invalid scope exception.

Reply:
Sounds to me like a bug of `FormatScopes()` not using the right delimiter. The scopes should be a list, and then the handler should join them together in a way that makes the service accept them.

Reply:
Changed to the recommended codes.